src/flavors/central-puget-sound/config.yml

```diff
@@ -494,18 +494,12 @@ place:
   location_item_name: location
   default_basemap: satellite
 
-  list:
-    fields:
-      # TODO: Add more field types here, and render html field types,
-      # depending on the category (same as the PWI flavor)
-      - name: classroom-learning
-        type: description
-
   #### begin dynamic form config ####
   place_detail:
     - category: featured_school
       admin_only: true
       includeOnForm: true
+      includeOnList: true
```

Comment:
At some point we normalized config keys to be `snake_case`; not sure how important it is to keep up that convention, but we might want to consider refactoring these new config keys to be snake case.

Reply:
I was trying to keep consistency with `includeOnForm`. I can switch it to snake case.

Reply:
Ah, yeah it might not be completely consistent.

Reply:
Since it's already inconsistent, and we are planning to move to JSON anyway, I'll just leave this camel cased.